ui/app/adapters/pki/issuer.js

```diff
@@ -21,6 +21,22 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
     }
   }
 
+  createRecord(store, type, snapshot) {
+    const { record, adapterOptions } = snapshot;
+    let url = this.urlForQuery(record.backend);
+    if (adapterOptions.import) {
+      url = `${url}/import/bundle`;
+    } else {
+      // TODO WIP generate
+      const type = 'root' || 'generate';
+      // record.type is internal or exported
```

Comment:
This is the pattern I used for generating vs importing a pki key [here](https://github.com/hashicorp/vault/blob/main/ui/app/adapters/pki/key.js#LL14)